src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMassesGenerator.h

```diff
@@ -74,19 +77,55 @@ namespace OpenMS
     void generateKnockoutDeltaMasses();
 
     /**
-     * @brief write the list of labels for each sample
+     * @brief write the list of labels for each of the sample
      */
-    void printLabelsList() const;
+    void printSamplesLabelsList() const;
```

Comment:
what is a sample in this context? I think it is not clear for the reader what is meant at this point or what the expected output might look like